content/glossary/sql/inner-join.md

```diff
@@ -0,0 +1,41 @@
+<!--
```

Comment:
remove "sql" from tags and subjects, title from markdown

Reply:
👍🏼